README.md

```diff
@@ -106,6 +109,74 @@ You can also keep the column `created_at` if you'd like.
 
 Hit save and then your table should be created.
 
+### Create a `customers` table with automattic population
+
+- Go to `Database` > `Tables`
+- Click `New Table`
+- Enter the name as `customers` and check `Enable Row Level Security (RLS)`
+- Configure the following columns:
+
+| Name                 | Type   | Default value | Primary |
+| -------------------- | ------ | ------------- | ------- |
+| `user_id`            | `uuid` | `auth.uid()`  | `true`  |
+| `stripe_customer_id` | `text` | `NULL`        | `false` |
+| `is_subscribed`      | `bool` | `false`       | `false` |
+
+- Click the link symbol next to the `user_id` column name, select schema `auth`,
+  table `users`, and column `id`. Now the `user_id` will link back to a user
+  object in Supabase Auth.
+- Next, go `Database` > `Functions` and click `Create a new function` with the
+  following values:
+  - Name of function = `create_customer`
+  - Schema = `public`
+  - Return type = `trigger`
+  - Definition =
+
+```sql
+begin
+  insert into public.customers(user_id) 
+  values (new.id);
+  return new;
+end;
+```
+
+- Click `Show advanced settings` and select `SECURITY DEFINER` under
+  `Type of security`
+- Click `Confirm`
+- Go to `Database` > `Triggers`
+- Click `Create a new trigger` and enter the following values:
+  - Name of trigger = `new_customer`
+  - Table = `users auth`
+  - Events = `Insert`
+  - Trigger type = `After the event`
+  - Orientation = `Row`
+  - Function to trigger = `create_customer`
+
+### Automate Stripe subscription updates via Supabase
```

Comment:
Using Database Webhooks is definitely a good way to keep things in sync in realtime, however it might not be required.

What I would do instead is create a method `createOrRetrieveCustomer` that takes in the user id and returns the stripe customer id if the customer already exists in Stripe, or creates a new stripe customer, stores the stripe customer id in the database, and then returns the stripe customer id.

You can see an implementation of that approach here: https://github.com/vercel/nextjs-subscription-payments/blob/main/utils/supabase-admin.ts#L52-L83

Reply:
Excellent suggestion! I've gone full circle and reverted webhooks for functionality relating to the `customers` table, and now it's looking much simpler. Thank you.